README.md

```diff
@@ -22,4 +22,54 @@ Testing is done via [pytest](pytest.org):
 
     pytest
 
-All tests should pass before landing on `master`.
\ No newline at end of file
+All tests should pass before landing on `master`.
+
+# Client<->Server Schema
+
+```
+Place: {
+    version: #,
+    id: "<yelp-id>",
+
+    name: "",
+    description: {
+        provider: "wikipedia",
+        text: ""
+    },
+    url: "",
+    phone: "", // consistent format
+
+    address: "",
+    coordinates: {
+        lat: #,
+        lng: #
+    },
+    categories: [{
+        id: "",
+        text: ""
+    }],
+
+    providers: {
+        yelp: {
+            rating: #, // normalized 0-5 w/ half values
+            totalReviewCount: #,
+            url: ""
+        } // ...
+    },
+
+    images: [{
+        src: "", // url to image
+        provider: "", // name to attribute on client
+        providerURL: "" // e.g. click target for image
+    }],
+
+    hours: {
+        monday: [
+            ["6:30", "18:00"]
+        ],
```

Comment:
I find Yelp's format usable but lacking – it'd be great to just index right into the hours for today (rather than having to search through a list of objects).

@jhugman Do you happen to know how Yelp handles the case that a restaurant is open for lunch, closed between lunch & dinner, and open again for dinner? That was another reason I liked Factual's style.